aya/src/programs/cgroup_device.rs

```diff
@@ -1,7 +1,7 @@
 //! Cgroup device programs.
 
 use crate::util::KernelVersion;
-use std::os::fd::{AsRawFd, RawFd};
+use std::os::fd::AsRawFd;
```

Comment:
```suggestion
use std::os::fd::AsRawFd as _;
```

Reply:
This trait is explicitely used in the signature for `CgroupDevice::attach` so it cannot be aliased to `_`.